indra/llrender/llimagegl.cpp

```diff
@@ -1052,7 +1052,7 @@ U32 type_width_from_pixtype(U32 pixtype)
 bool should_stagger_image_set(bool compressed)
 {
 #if LL_DARWIN || LL_LINUX
-    return false;
+    return !compressed && on_main_thread() && gGLManager.mIsAMD;
```

Comment:
This change was necessary on Linux, from what I can remember, this was causing an issue with apputil that @RunitaiLinden helped me with. Davep, do you by chance remember why we made this change? IIRC, something was really slow prior to making this change -- we were spending a lot of time around here.

cc: @brad-linden

Reply:
here's the commit: https://github.com/secondlife/viewer/pull/2353/commits/0f90b5698557a3045064728df422ba38cd3ab30a

Reply:
yeah I just automatically resolved the conflict here, but now that you mention it, perhaps the better solution would be to spit out these cases now:
```suggestion
#if LL_LINUX
    return false;
#elif LL_DARWIN
    return !compressed && on_main_thread() && gGLManager.mIsAMD;
#else
```